tests/Feature/MyTests/NoteTest.php

```diff
@@ -0,0 +1,159 @@
+<?php
+
+namespace Tests\Feature;
+
+// use Illuminate\Foundation\Testing\RefreshDatabase;
+use App\Models\Note;
+use App\Models\User;
+use Egulias\EmailValidator\Parser\Comment;
+use Egulias\EmailValidator\Parser\CommentStrategy\LocalComment;
+use Illuminate\Foundation\Testing\RefreshDatabase;
+use PHPUnit\Framework\SkippedTest;
+use Tests\TestCase;
+
+/*
+    Tests will follow the AAA (Arange Act Assert) format
+*/
+
+class NoteTest extends TestCase
+{
+    use RefreshDatabase; //clears the database for each test run
+    protected $userOne, $userTwo;
+
+    protected function setUp(): void
+    {
+        parent::setUp();
+        // Create 2 test users, need 2 so we can test note viewing authentication
+        $userData = [];
+
+        for ($i = 1; $i <= 2; $i++) {
+            $userData[] = [
+                'name' => 'Test User #' . $i,
+                'email' => 'testemail' . $i . '@test.com',
+                'password' => bcrypt('pass123.'),
+            ];
+        }
+
+        User::factory()->createMany($userData);
+        $this->userOne = User::find(1); //assign userOne
+        $this->userTwo = User::find(2);//assign userTwo
+
+        //Prefab'd notes for user using different looping method than user prefab loop
+        Note::factory(10)->create()->each(function ($note) {
+            $note->update([
+                'note' => 'this is note #' . $note->id . ' for user #' . $note->user_id,
+            ]);
+        });
+
+    }
+
+    public function test_application_response_if_user_isnt_logged_in()
+    {
+
+        $response = $this->get('/');
+        $response->assertStatus(302);
+
+        $response = $this->get('/login');
+        $response->assertStatus(200);
+    }
+
+    public function test_application_response_if_user_is_logged_in()
+    {
+        $this->actingAs($this->userOne);
+        $response = $this->get('/note');
```

Comment:
Earlier (in the `isnt_logged_in` test case), `/` and `/login` are tested. Here, `/note` is tested. I think it would be more instructive if `/note` were tested in the `isnt_logged_in` testcase to show the difference.

Reply:
Ah I see, I understand how that would make more sense since the homepage of the application is `/note`. It would've been better to test that `/ntoe` redirects if the user isn't logged in rather than testing `/` since it will redirect on `/` whether or not the user is logged in.